parent/pom.xml

```diff
@@ -241,11 +241,12 @@
 					<!-- set useUIThread=false for swtbot tests -->
 
 					<includes>
-						<include>**/AllTests.class</include>
-						<include>**/*AllTests*.class</include>
-						<include>**/*AllBotTests*.class</include>
-						<include>**/*TestSuite*.class</include>
+						<include>**/*Test*.class</include>
```

Comment:
including *Test* would be a bad thing siince it executes the testsuites then too...would it not ?